### ISSUE TITLE
Add Skip Link

### ISSUE THREAD

**Issue by kaitlinbolling** [alleyinteractive/create-wordpress-theme#43]
### Description

We should add a "Skip to Content" link that allows users to skip directly to the main content area on each page. This link should be visible to screen readers, as well as to sighted users when it is focused, so it should be styled appropriately.

### Use Case

As a user viewing the site with a keyboard or other assistive device, I would like to be able to quickly skip to the main content of the page.

**Comment by kaitlinbolling** [comment]
WordPress core automatically inserts a skip link into block themes that include the `<main>` tag. We can use core's implementation to start.

See: https://developer.wordpress.org/reference/functions/wp_enqueue_block_template_skip_link/
